### ISSUE TITLE
Information Assurance: resolve `plexus-archiver` vulnerabilities

### ISSUE THREAD

**Issue by d-ryan-ashcraft** [TechnologyBrewery/fermenter#33]
The following `plexus-archiver` vulnerabilities (CVE-2023-37460 & CVE-2018-1002200)

<img width="925" alt="image" src="https://github.com/TechnologyBrewery/fermenter/assets/118617093/a0ce76ae-c46a-4631-80fe-aa1fbcfe61ec">

Currently, `mvn dependency:tree` resolves to version 2.2 and we need at least 4.8.
```
[INFO] +- org.apache.maven.plugin-testing:maven-plugin-testing-harness:jar:3.3.0:test
[INFO] |  \- org.codehaus.plexus:plexus-archiver:jar:2.2:test
[INFO] |     +- org.codehaus.plexus:plexus-container-default:jar:1.0-alpha-9-stable-1:test
[INFO] |     |  \- classworlds:classworlds:jar:1.1-alpha-2:test
[INFO] |     \- org.codehaus.plexus:plexus-io:jar:2.0.4:test
```

**Comment by d-ryan-ashcraft** [comment]
Resolved - `mvn dependency:tree` now resolves to 4.8:
```
[INFO] +- org.apache.maven.plugin-testing:maven-plugin-testing-harness:jar:4.0.0-alpha-1:test
[INFO] |  +- org.apache.maven:plexus-utils:jar:4.0.0-alpha-4:test
[INFO] |  +- org.codehaus.plexus:plexus-testing:jar:1.1.0:test
[INFO] |  |  \- org.junit.jupiter:junit-jupiter-api:jar:5.8.0:test
[INFO] |  |     +- org.opentest4j:opentest4j:jar:1.2.0:test
[INFO] |  |     +- org.junit.platform:junit-platform-commons:jar:1.8.0:test
[INFO] |  |     \- org.apiguardian:apiguardian-api:jar:1.1.2:test
[INFO] |  \- org.mockito:mockito-core:jar:4.6.1:test
[INFO] |     +- net.bytebuddy:byte-buddy:jar:1.12.10:test
[INFO] |     +- net.bytebuddy:byte-buddy-agent:jar:1.12.10:test
[INFO] |     \- org.objenesis:objenesis:jar:3.2:test
[INFO] +- org.codehaus.plexus:plexus-archiver:jar:4.8.0:test
[INFO] |  +- org.codehaus.plexus:plexus-io:jar:3.4.1:test
[INFO] |  +- org.apache.commons:commons-compress:jar:1.23.0:test
[INFO] |  +- org.iq80.snappy:snappy:jar:0.4:test
[INFO] |  +- org.tukaani:xz:jar:1.9:test
[INFO] |  \- com.github.luben:zstd-jni:jar:1.5.5-5:test
```

**Comment by d-ryan-ashcraft** [comment]
CI build passed:
![image](https://github.com/TechnologyBrewery/fermenter/assets/118617093/1e28b759-3a55-49e4-b241-9929b902940b)
